.gitignore

```diff
@@ -13,7 +13,6 @@ wp-content/advanced-cache.php
 wp-content/wp-cache-config.php
 sitemap.xml
 sitemap.xml.gz
-*.log
```

Comment:
Why was this removed? I'm neutral on this change, just wondering.

Reply:
duplicated below (line 42). Originally opened in https://github.com/pantheon-systems/WordPress/pull/219, just been sitting for a while.